TAO/orbsvcs/tests/CosEvent/Basic/Random.cpp

```diff
@@ -344,6 +344,12 @@ RND_Consumer::push (const CORBA::Any &event)
 }
 
 void
+RND_Timer::push (const CORBA::Any &event)
```

Comment:
Why isn't this one in the header? This operation shouldn't be needed, maybe report a bug to Oracle?